encryption_works.md

```diff
@@ -98,25 +98,41 @@ Tails, which is a live DVD and live USB GNU/Linux distribution that I will discu
+> The good news is that they don't appear to have any exploit on Tor or on Tor Browser. The other good news is that their proposed attack doesn't scale well. They need to put malicious hardware on the local network of their target user, which requires choosing their target, locating her, and then arranging for the hardware to arrive in the right place. So it's not really practical to launch the attack on many Tor users at once.
 
-![](https://raw.github.com/micahflee/encryption-works/master/images/torbrowser.png)
+As with all privacy-enhancing software, an attacker with physical access to the device can compromise you in any number of ways.
 
-When Snowden [was answering questions on Guardian's website](http://www.guardian.co.uk/world/2013/jun/17/edward-snowden-nsa-files-whistleblower) from a "secure Internet connection", he was probably routing his traffic through the Tor network. He may have also been using a [bridge](https://bridges.torproject.org/) to connect to the Tor network to make the fact that he was using Tor from his IP address less obvious to eavesdroppers.
+### Setup
+
+The easiest way to start using Tor is to download and install the [Tor Browser Bundle](https://www.torproject.org/download/download-easy.html.en). 
+
+### Browsing Habits
+
+You should use Tor differently than you use a regular browser: some of the features it uses to obscure your location can be undermined if you don't take a few extra steps to protect yourself.
+
+- Don't enable or install browser plugins, including Adobe Flash. These can bypass Tor's security and give away your location.
+- Be aware of unencrypted sites, which don't have a lock icon next to the URL. Tor anonymizes your Internet traffic but unencrypted connections can still be eavesdropped on between the final node and the Internet server.
+- Don't download documents downloaded through Tor, since these documents can reveal your regular IP address to an adversary. Open them while offline or consider using Tails.
+
+As with Tails, these limitations are designed to keep Tor as secure as possible. They can get in the way of your regular browsing habits, and so you should keep another web browser around (with add-ons such as [HTTPS Everywhere](https://www.eff.org/https-everywhere)) for other browsing.
+
+### Theoretical Shortcomings
+
+In theory, it’s possible to de-anonymize Tor traffic by studying the timing of packets as they move across the network, but it’s too early to say whether large intelligence agencies can defeat the anonymity of the Tor network. Even so, an attack like this becomes much harder as the number of nodes and the volume of traffic increases. If you're a student or staff-member at a college or university, the Electronic Frontier Foundation encourages you to [set up](https://www.eff.org/torchallenge/tor-on-campus.html) a Tor node on campus.
```

Comment:
Again, we don't want to minimize the possible vulnerabilities in Tor. I would move up the part about running a node to under the sentence about global adversaries and get rid of "theoretical shortcomings"